anomdetect/src/test/java/com/expedia/adaptivealerting/anomdetect/AnomalyToMetricMapperTest.java

```diff
@@ -28,6 +29,7 @@
 import static com.expedia.adaptivealerting.anomdetect.AnomalyToMetricMapper.AA_DETECTOR_UUID;
 import static org.junit.Assert.assertTrue;
 
+@Slf4j
```

Comment:
is this required?

Reply:
Yeah, I'm displaying an error message if there's an exception. I added this after your question.